### PR TITLE
Requesting a pull to mozilla:master from k88hudson:t2522

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -209,19 +225,45 @@ <h5 class="title">
...
-      $('#projects .carousel-items').jCarouselLite({
-        btnNext: '.right',
-        btnPrev: '.left',
-        circular: true,
-      });
-    });
+
+    (function() {
+      var IFRAME_WIDTH = 560,
+          IFRAME_HEIGHT = 340,
```
Comment (mjschranz) [inline public/index.html]: These don't appear to be used for anything.

File: public/index.html [hunk 2]
```diff
@@ -26,11 +26,13 @@
                 </a>
               </div>
               <a href="http://www.mozilla.org/" id="tabzilla">Mozilla</a>
+              <!--
               <nav id="nav-main" role="navigation">
                 <ul>
                   <li class="home"><a href="/">Home</a></li>
                 </ul>
               </nav>
+              -->
```
Comment (mjschranz) [inline public/index.html]: If you don't need these elements, might as well remove them.

Comment (mjschranz) [inline public/index.html]: You missed this one, unless there is a reason you are keeping this one here.

File: public/index.html [hunk 3]
```diff
@@ -209,19 +222,47 @@ <h5 class="title">
...
+        newiFrame.src = this.querySelector( "img" ).getAttribute( "alt" );
+
+        modalInner.appendChild( newiFrame );
+        modal.style.opacity = 1;
+        modal.style.visibility = "visible";
+
+        modal.addEventListener( "click", function closeModal() {
+          this.style.opacity = "";
+          this.style.visibility = "";
+          console.log( modalInner );
```
Comment (mjschranz) [inline public/index.html]: Rogue console.log

File: public/index.html [hunk 4]
```diff
@@ -67,99 +67,112 @@
...
+                    <li>Uses soundcloud</li>
+                    <li>Other stuff</li>
                   </ul>
                 </div>
               </div>
-              <a class="more" href="#">See all projects...</a>
+              <div class="span5">
+                <h4><a href="/templates/basic/?savedDataUrl=tour.json">Lego</a></h4>
+                  <div class="thumbnail preview-thumbnail">
+                    <img src="http://dl.dropbox.com/u/29163874/Screenshots/7m.png" alt="http://dev.butyr.org/v/gh_.html">
```
Comment (mjschranz) [inline public/index.html]: Technically I suppose there isn't anything wrong with using alt this way but as far as I know it isn't intended.

Why not just use your own attribute on these like, data-iframe-url or something instead? That way it's more obvious what the intent is.

File: public/index.html [hunk 5]
```diff
@@ -15,151 +15,140 @@
...
+			         </p>
             </div>
-            <div class="span4">
-              <video controls preload="none" width="344" height="232"
-                poster="media/img/poster.png">
-                  <source src="http://videos.mozilla.org/serv/webmademovies/popcornsite.webm" type="video/webm">
-                  <source src="http://videos.mozilla.org/serv/webmademovies/popcornsite.mp4" type="video/mp4">
-              </video>
+            <div class="span5">
+              <iframe class="iframe-template" src='http://dev.butyr.org/v/gh_.html' width='560' height='340' mozallowfullscreen webkitallowfullscreen allowfullscreen></iframe>
```
Comment (secretrobotron) [inline public/index.html]: Mixing quotes.

File: public/index.html [hunk 6]
```diff
@@ -15,151 +15,140 @@
...
+                <div class="span10">
+                  <h3>Remix and explore these great projects</h3>
+                </div>
+            </div>
+            <div class="row">
+              <div class="span10">
+                <h4><a href="/templates/basic/?savedDataUrl=tour.json">Neighbourhood Tour</a></h4>
+              </div>
+              <div class="span7">
+                <iframe class="iframe-template" src='http://dev.butyr.org/v/gh_.html' width='560' height='340' mozallowfullscreen webkitallowfullscreen allowfullscreen></iframe>
```
Comment (secretrobotron) [inline public/index.html]: Mixing quotes.

File: public/index.html (hunk 4 above)
Comment (secretrobotron) [inline public/index.html]: I agree with Matt.

File: public/index.html [hunk 7]
```diff
@@ -209,19 +198,47 @@ <h5 class="title">
...
-    });
+
+    (function() {
+      var modal = document.querySelector( "#modal-overlay" ),
+          modalInner = modal.querySelector( "#modal" ),
+          iframeTemplate = document.querySelector( ".iframe-template" ),
+          previews = document.querySelectorAll( ".preview-thumbnail" ),
+          i, l;
+
+      function spawniFrame( e ) {
```
Comment (secretrobotron) [inline public/index.html]: Fix your camelCase in `spawniFrame`.

File: public/index.html [hunk 8]
```diff
@@ -209,19 +198,47 @@ <h5 class="title">
...
+        var newiFrame = iframeTemplate.cloneNode( true );
+        newiFrame.src = this.querySelector( "img" ).getAttribute( "alt" );
```
Comment (secretrobotron) [inline public/index.html]: See https://github.com/mozilla/butter/pull/1147/files#L2R93

File: public/media/css/page.css [hunk 9]
```diff
@@ -268,13 +321,71 @@ Home styles
...
+  float: left;
+  width: 260px;
+}
+
+#bottom-section {
+  padding-top: 48px;
+}
+
+#bottom-section img {
+  display: none; /* for now */
```
Comment (secretrobotron) [inline public/media/css/page.css]: Does this need to be here? I fear this and the actual images in the html will be left in without further consideration.

File: public/index.html [hunk 10]
```diff
@@ -16,150 +16,128 @@
...
-                        <div class="caption">
-                          <p>by Mozilla</p>
-                          <h5 class="title"><a href="/templates/basic/?savedDataUrl=meme.json">Video Meme</a></h5>
-                        </div>
-                      </div>
-                    </li>
-                  </ul>
+          <div id="projects">
+            <div class="row">
+                <div class="span10">
```
Comment (secretrobotron) [inline public/index.html]: Indentation

**Pull request merged**